ubuntu/14.04/standard/Dockerfile

```diff
@@ -117,6 +117,11 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
     powershell \
  && rm -rf /var/lib/apt/lists/*
 
+# Install Mysql
+RUN apt-get update \
+ && apt-get install mysql-client -y
```

Comment:
We need to be sure and add "&& rm -rf /var/lib/apt/lists/*" to this set of commands, like the others, to reduce the size of the image's individual layers.

Reply:
Also, I've been trying to keep the various tool installations loosely ordered alphabetically (although I just noticed Open**J**DK and **G**o aren't following this). Feel free to commit as-is and when I sync and update the README.md, I'll fix the ordering of OpenJDK and Go as well as putting MySQL right before the .NET SDK. Thanks!

Reply:
@thechrisjohnson  added "&& rm -rf /var/lib/apt/lists/*" it .Also tested . Please review and merge it .